README.md

```diff
@@ -17,11 +17,10 @@ First add credentials to a .env file, which you won't commit. If you track this
 ```env
 // .env.production
 ALGOLIA_APP_ID=XXX
-ALGOLIA_API_KEY=XXX
 ALGOLIA_INDEX_NAME=XXX
 ```
 
-```js
+```javascript:title=gatsby-config.js
```

Comment:
I don't think this syntax works on GitHub

Reply:
done.